flink-core/src/main/java/org/apache/flink/util/FlinkUserCodeClassLoaders.java

```diff
@@ -112,7 +110,7 @@ public static ResolveOrder fromString(String resolveOrder) {
      */
     public static class ParentFirstClassLoader extends FlinkUserCodeClassLoader {
```

Comment:
Let's mark this class as `@Internal` because currently we do not expect users to use it.